apps/web/components/sections/project/project-updates.tsx

```
@@ -0,0 +1,54 @@
+					<h2 className="text-xl font-semibold text-black">Success Gallery</h2>
+					<a
+						href="#"
+						className="text-sm font-medium text-blue-600 hover:underline"
+					>
+						View All
+					</a>
+				</div>
+				<div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-4">
+					{successGalleryItems.map((item, index) => (
+						<div
+							key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)